### PR TITLE
Update Event Grid sample role assignment ID generation

### PR DESCRIPTION
**Pull request by johndowns**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Update the seed values for `uniqueString()` in the `101-event-grid` sample.

### CODE REVIEW

**Comment by johndowns** [comment]
Closing and re-opening to retry the USGov deployment

**Comment by johndowns** [comment]
@bmoore-msft Any ideas about the failing USGov test deployment?

File: 101-event-grid/prereqs/prereq.azuredeploy.bicep [hunk 1]
```diff
@@ -144,7 +144,7 @@ resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-
 
 resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
   scope: functionApp
-  name: guid(resourceGroup().id, managedIdentityName)
+  name: guid(resourceGroup().id, managedIdentity.id)
```
Comment (bmoore-msft) [inline 101-event-grid/prereqs/prereq.azuredeploy.bicep:147]: guid should be seeded with scope, principal and role, e.g.

guid(functionApp.id, managedIdentity.id, contributorRoleDefinitionId)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-event-grid/prereqs/prereq.azuredeploy.bicep (hunk 1 above)
Comment (johndowns) [reply 101-event-grid/prereqs/prereq.azuredeploy.bicep]: Fixed.

**Pull request merged**